protocol.c

```diff
@@ -116,48 +116,49 @@ void protocol_main_loop()
         iscomment = false;
         char_counter = 0;
       } else {
-        if (iscomment) {
-          // Throw away all comment characters
-          if (c == ')') {
-            // End of comment. Resume line.
-            iscomment = false;
+          if (iscomment) {
+            // Throw away all comment characters
+            if (c == ')') {
+              // End of comment. Resume line.
+              iscomment = false;
+            }
```

Comment:
This looks like you just changed the indentation of the code without changing anything significant. Is that correct? If so, undo it; large changes to the whitespace formatting should go in a separate PR that doesn't also have changes to functionality in it. 

This comment applies to lines 119-158ish.